src/frontend/src/redux/system/actions.ts

```diff
@@ -37,9 +38,19 @@ export function logOut(): any {
   };
 }
 
-export function refreshLocalAuth() {
-  return {
-    type: LOG_IN
+
+export function refreshLocalAuth(): any {
+  return (dispatch: Dispatch) => {
+    return apiService(dispatch, {
```

Comment:
This post request is missing a body which for this endpoint should normally be email/password. But we don't hold onto the password in the frontend so there's no way to quietly use call method. 

Agreeing with Kent: it's not clear why this backend call is required

Reply:
I don't like the phrasing "agreeing with Kent" as it makes it sound like I was making a rhetorical question, which I try hard not to do in a PR review. If you were a student in a classroom and a fellow student asked the professor "what is a land-value tax?", it would be odd to say "I agree with that question". This is a fuzzy line but I want to keep the distinction between "Why do we need to hit the backend again here?" versus "I don't think we need to hit the backend again here".

Reply:
Fair. i did mean, "I have the same questions as Kent"

Reply:
But now that you believe we don't need a backend call...

Reply:
I'm honestly don't know. I naively tried eliminating the the calling code and I didn't figure out a way to get the persist feature to work, so I just don't know what's going on or required.

Reply:
Sounds good; I guess we'll wait for a reply :P